.github/workflows/gradle-release.yml

```
@@ -77,3 +77,161 @@ jobs:
       - name: Git Push
         run: |
           git push
+name: Gradle Release
```

actionlint (1.7.4)

key "name" is duplicated in workflow. previously defined at line:1,col:1

(syntax-check)